### PR TITLE
Buyer notification created

### PR DESCRIPTION
**Pull request by kumarpawan1**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by gjoshevski** [comment]
What happens if the MarketplaceSellerEmail is not confirmed?

**Comment by chotafakir** [comment]
Why is the test listing (WUPHF.com) and test email data hard-coded in the script "src/register-new-subscriber.js"? Will it be updated as an input parameter in the main template?

**Review by chotafakir — COMMENTED** [review]
The customer name and ISV name should be dynamic and pull the information from landing page submission data and product listing. The hard-coded email with dummy data will require editing the script before launching the stack, and that defeats the purpose of 1-click launch.

File: template.yaml [hunk 1]
```diff
@@ -45,14 +45,17 @@ Parameters:
   MarketplaceTechAdminEmail:
     Type: String
     AllowedPattern: ".*"
+  
+  MarketplaceSellerEmail:
```
Comment (gjoshevski) [inline template.yaml:49]: This parameter should be optional. What if I want to use my own service for sending emails?

File: src/register-new-subscriber.js [hunk 2]
```diff
@@ -65,11 +90,10 @@ exports.registerNewSubscriber = async (event) => {
               }`,
           QueueUrl: entitlementQueueUrl,
         };
-
         await sqs.sendMessage(SQSParams).promise();
       }
-
-      return lambdaResponse(200, 'Success! Registration completed. You have purchased an enterprise product that requires some additional setup. A representative from our team will be contacting you within two business days with your account credentials. Please contact Support through our website if you have any questions.');
+       await setBuyerNotificationHandler(contactEmail);
```
Comment (gjoshevski) [inline src/register-new-subscriber.js:95]: This step should be performed only if MarketplaceSellerEmail is provided. MarketplaceSellerEmail should be optional.

File: README.md [hunk 3]
```diff
@@ -11,6 +11,7 @@ If you are a new seller on AWS Marketplace, we advise you to check the following
 * [SaaS Integration Video](https://www.youtube.com/watch?v=glG44f-L8us) : This video guides you through the requirements and steps needed to integrate. 
 * [SaaS Pricing Video](https://www.youtube.com/watch?v=E0uWp8nhzAk) : This video guides you through the pricing options available when choosing to list a SaaS product.
 * [AWS Marketplace - Seller Guide](https://docs.aws.amazon.com/marketplace/latest/userguide/what-is-marketplace.html) : This document covers more information about creating a SaaS product, pricing, and setting up your integration.
+* [Verify an email address](https://docs.aws.amazon.com/ses/latest/DeveloperGuide/verify-email-addresses-procedure.html): This document gives step by step instructions to verify email address that will be used as your SELLERSESVERIFIEDEMAILADDRESS address later)
```
Comment (gjoshevski) [inline README.md:14]: This does not feel like the right place to provide this link. All of the above links are AWS Marketplace related and intended to help the users understand AWS Marketplace. Please provide this instructions and explanation in the deployment section.

File: src/register-new-subscriber.js [hunk 4]
```diff
@@ -15,6 +15,31 @@ const lambdaResponse = (statusCode, body) => ({
   body: JSON.stringify(body),
 });
 
+let setBuyerNotificationHandler = function (contactEmail) {
+  var params = {
```
Comment (gjoshevski) [inline src/register-new-subscriber.js:19]: to be consistent with the coding standard, please do not use var.

File: template.yaml [hunk 5]
```diff
@@ -224,16 +228,20 @@ Resources:
...
-        - !If
-          - CreateEntitlementLogic
-          - Statement:
-            - Sid: SQSPolice
-              Effect: Allow
-              Action:
-                - sqs:SendMessage
-              Resource: !GetAtt EntitlementSQSQueue.Arn
-          - !Ref "AWS::NoValue"
+        - Statement:
```
Comment (gjoshevski) [inline template.yaml:231]: Why is this overwriting the !If statment?

**Review by gjoshevski — CHANGES_REQUESTED** [review]

**Review by gjoshevski — APPROVED** [review]

**Pull request merged**